HW-0.Rmd

```diff
@@ -21,9 +21,9 @@ completion. Your analysis is now online! You can update what's online by
 repeating steps 2 and 3.
 1. Copy the URL of the resulting RPubs page.
 1. Replace the https://www.nhl.com/ URL below with the URL of your RPubs page:  
-[My RPubs Document](https://www.nhl.com/)
+[My RPubs Document] (http://rpubs.com/tsingh/HW-0)
```

Comment:
Looks good, except next time don't have a space between the `[]` and the `()`. The way you quickly include HTML links in markdown format is `[My RPubs Document](http://rpubs.com/tsingh/HW-0)` without the space.